apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -25,6 +34,23 @@ export default class ReviewTab extends Component {
           token: request.getResponseHeader('csrf-token')
         });
       });
+
+    codeReviewDataApi
+      .getPeerReviewStatus(channelId, serverLevelId, serverScriptId)
+      .done(data => {
+        const id = (data && data.id) || null;
+        this.setState({
+          reviewCheckboxEnabled: data.canMarkReviewable,
+          isReadyForReview: data.reviewEnabled,
+          reviewableProjectId: id
+        });
+      })
+      .fail(() => {
```

Comment:
in the failure case should we instead show an error message?

Reply:
Hmm good question, I'll double check for what the intended experience should be. Currently if this call fails, then we just don't display the Enable Peer Review checkbox.

Reply:
Added a catch-all for adding error messaging in this feature: https://codedotorg.atlassian.net/browse/CSA-585